upgrading/downgrade.adoc

```diff
@@ -118,22 +105,17 @@ remove the following packages:
 # systemctl reboot
 ----
 
-[[downgrade-docker]]
-== Downgrading Docker
-
-Both {product-title} 3.9 and 3.10 require Docker 1.13, so Docker does not need
-to be downgraded.
 
 [[downgrade-reinstalling-rpms]]
 == Reinstalling RPMs
 
-. Disable the {product-title} 3.10 repositories, and re-enable the 3.9
+. Disable the {product-title} 3.11 repositories, and re-enable the 3.10
 repositories:
 +
 ----
 # subscription-manager repos \
-    --disable=rhel-7-server-ose-3.10-rpms \
-    --enable=rhel-7-server-ose-3.9-rpms
+    --disable=rhel-7-server-ose-3.11-rpms \
+    --enable=rhel-7-server-ose-3.10-rpms
 ----
 
 . On each master, install the following packages:
```

Comment:
Following list need to be updated.
atomic-openshift-node-3.10.51-1.git.0.90663d3.el7.x86_64
atomic-openshift-docker-excluder-3.10.51-1.git.0.90663d3.el7.noarch
atomic-openshift-excluder-3.10.51-1.git.0.90663d3.el7.noarch
atomic-openshift-clients-3.10.51-1.git.0.90663d3.el7.x86_64
atomic-openshift-hyperkube-3.10.51-1.git.0.90663d3.el7.x86_64
atomic-openshift-3.10.51-1.git.0.90663d3.el7.x86_64

Following sections on each node need to be updated too.

Reply:
when re-install old version openshift, master and node should install the same pkgs, so suggest to combine "On each master, install the following packages:" and "On each node, install the following packages:"

Reply:
LGTM now